care/facility/migrations/0102_auto_20200423_1529.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 2.2.11 on 2020-04-23 09:59
+
+from django.db import migrations, models
+import uuid
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [standard import "import uuid" should be placed before "from django.db import migrations, models"](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5408178)